### ISSUE TITLE
Fix bug: fetchOrganizationData is executed twice

### ISSUE THREAD

**Issue by nya-elimu** [elimu-ai/aragon-dao-webapp#10]
How to reproduce:
```
$ node app.js 
Server running at http://127.0.0.1:3000/
fetchOrganizationData
org: [object Object]
apps: [object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object]
fetchOrganizationData
org: [object Object]
apps: [object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object]
```

**Comment by nya-elimu** [comment]
@repoman Any theories to why this is happening?

**Comment by repoman** [comment]
> @repoman Any theories to why this is happening?

I think the second request is the browser trying to get the favicon. And then that request is handled by the same server http code, so `fetchApps` is run. It can be fixed by making a own handler for the favicon, or using express with some routing setup so that certain requests use certain handlers.

**Comment by nya-elimu** [comment]
Was fixed as part of #11. Closing.
